pytorch/training/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -169,7 +169,6 @@ RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value \
     typing \
     h5py \
     requests \
-    future \
```

Comment:
Did you find a solution for addressing "future" package CVE other than adding the vulnerability id to ignore list?

Reply:
future package comes with python as module already, not need to install it